src/RegExLib.Core/Entities/Author.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using RegExLib.Core.Events;
+using RegExLib.Core.Interfaces;
+using RegExLib.SharedKernel;
+
+namespace RegExLib.Core.Entities
+{
+    public class Author : BaseEntity
+    {
+        public string FullName { get; set; }
+        public string Username { get; set; }
+        public string UserId { get; set; }
+        public List<Expression> Expressions { get; private set; } = new List<Expression>();
+
+        public Author()
+        {
+        }
+
+        public Author AddExpression(Expression expression)
+        {
+            if (expression != null)
+            {
+                Expressions.Add(expression);
+            }
+
+            return this;
+        }
```

Comment:
Do we need this to be a fluent API? Following [CQS](https://martinfowler.com/bliki/CommandQuerySeparation.html) instead might make the codebase easier to reason about.

Reply:
I am not sure what you want to change in that function.

Reply:
Basically, I'm suggesting that this be a `void`-returning method. Another way to think about this is through the lens of the [YAGNI ](https://www.martinfowler.com/bliki/Yagni.html) principle; It's currently written to supporting chaining multiple `AddExpression(...)` calls on an `Author` instance (a.k.a. "fluent" or "method chaining"), but I think it's best to keep it simple until there's actually a need to shape the public API for that scenario.

Reply:
Done 

```csharp
        public void AddExpression(Expression expression)
        {
            if (expression != null)
            {
                Expressions.Add(expression);
            }
        }
```